archlinuxcn/rustnet/PKGBUILD

```diff
@@ -0,0 +1,73 @@
+# Maintainer: DeepChirp <DeepChirp@outlook.com>
+
+pkgname=rustnet
+_pkgname=${pkgname%}
+_reponame=${pkgname%}
+pkgver=0.11.0
+pkgrel=3
+pkgdesc="Real-time network monitoring TUI with process identification via eBPF and deep packet inspection"
+arch=('x86_64' 'armv7h' 'aarch64')
+_author=domcyrus
+url="https://github.com/${_author}/${_reponame}"
+license=('Apache-2.0')
+depends=('libpcap' 'libelf' 'zlib' 'gcc-libs' 'glibc')
+makedepends=('git' 'cargo' 'pkgconf' 'clang' 'llvm' 'lld' 'libbpf')
+provides=(${pkgname%})
+conflicts=("${pkgname%}-git" "${pkgname%}-bin")
+options=(!debug strip)
```

Comment:
Why not debug?

Reply:
> Why not debug?

Because when I use `makepkg -si` locally, I find that it installs both normal packages and debug packages. Do I still need to add the package with `-debug` suffix in `package.list`?

Reply:
No. Debug packages are recognized automatically.